plugin/inc/Lazysizes/Lazysizes.php

```diff
@@ -271,6 +271,13 @@ public function rewrite_image_block( $block_content, $block, $instance ) {
 			return $block_content;
 		}
 
+		/**
+		 * Re-add attributes from the original image to the new image.
+		 */
+		foreach ( $block_image->attributes as $attribute ) {
+			$new_image_html = str_replace( '<img', '<img ' . $attribute->name . '="' . $attribute->value . '"', $new_image_html );
```

Comment:
Instead of using str_replace consider adding the attributes directly into the $attr array property of wp_get_attachment_image.

https://developer.wordpress.org/reference/functions/wp_get_attachment_image/